.github/actions/setup-node/action.yml

```diff
@@ -26,6 +26,7 @@ runs:
       with:
         node-version: 16
         cache: pnpm
+        check-latest: true
```

Comment:
for anyone else looking:
https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#check-latest-version

This will make CI slower. Were you seeing an issue with using a cached Node version?

Reply:
This is a unnecessary change and I will withdraw it